cli/lsp/README.md

```diff
@@ -159,3 +160,21 @@ client:
     suggestions: boolean;
   }
   ```
+
+## Language IDs
+
+The language server supports diagnostics and formatting for the following
+language IDs:
+
+- `"javascript"`
+- `"javascriptreact"`
+- `"jsx"` _non standard_
```

Comment:
Maybe add note that it's the same as `javascriptreact`?

Reply:
Because they are non-standard: https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentItem. I didn't want to get into what they do here, as I would prefer they not be used and it is an "internal" concern.

I will link to the standard.

Reply:
Actually, upon refelction I will do both. 👍